src/shared/components/youtube/index.js

```diff
@@ -1,43 +1,86 @@
-import React, { Component } from 'react'
+import React, { PureComponent } from 'react'
 import classNames from 'classnames'
 import Svg from 'shared/components/svg'
 import styles from './index.module.css'
 import playSvg from './media/play.sprite.svg'
+import ReactYoutube from 'react-youtube'
 
-class Youtube extends Component {
-  state = { clicked: false }
+const opts = {
```

Comment:
Why is this outside the component scope?

Reply:
It shouldn't. Thanks.